RadarSDKTests/RadarSDKTests.m

```diff
@@ -823,7 +823,7 @@ - (void)test_Radar_getContext_errorLocation {
         [expectation fulfill];
     }];
 
-    [self waitForExpectationsWithTimeout:30
+    [self waitForExpectationsWithTimeout:60
```

Comment:
adding extra time here as the gh action keeps timing out. This is probably made worst by us needing to use a newer  (and probably more demanding) iphone simulator as support for older simulators are being dropped.